benchmarks/aes/nodejs/package.json

```diff
@@ -2,15 +2,14 @@
   "name": "aes-nodejs",
   "version": "0.1.0",
   "dependencies": {
-    "@grpc/grpc-js": "1.1.3",
+    "@grpc/grpc-js": "^1.6.7",
```

Comment:
please fix all the versions, don't use "^". A transparently updated version might break the build and we won't be able to easily figure out which version broke it.

Reply:
`npm audit fix --force` does this on it's own, do you want me to change it?